src/wire/igmpv2.rs

```diff
@@ -0,0 +1,420 @@
+/// Internet Group Management Protocol v2
+/// defined in [RFC_2236]
```

Comment:
- I don't see the href for this link.
 - AFAIK if this was intended to be module level docs it should be prefixed with `//!`.
 - Why was IGMPv2 (RFC 2236) implemented instead of IGMPv3 (RFC 3376)?

Reply:
> I don't see the href for this link.

Thanks, I added it.

> AFAIK if this was intended to be module level docs it should be prefixed with //!.

Right! Only that the `igmpv2` mod isn't exported as-is but its individual exports are reexported by mod `wire`, therefore hiding any documentation at the `mod igmpv2` level. I decided to move this documentation chunk to the `IgmpPacket` struct.

> Why was IGMPv2 (RFC 2236) implemented instead of IGMPv3 (RFC 3376)?

I guess @podhrmic has done so for backwards compatibility. v3 can be added at any later time of course.